.storybook/themes/news.scss

```diff
@@ -1368,6 +1384,25 @@
 						),
 					),
 				),
+				large-manual-promo: (
+					components: (
+						stack: (
+							gap: var(--global-spacing-2),
```

Comment:
If the value does not change between default and desktop we don't need to add the same value here